js/ui/tag_box.js

```diff
@@ -990,9 +984,7 @@ var TagBox = SelectBox.inherit({
     },
 
     _refreshSelected: function() {
-        this._list && this._suppressingSelectionChanged(function() {
-            this.callBase();
-        });
+        this._list && this._list.option("selectedItems", this._selectedItems);
```

Comment:
Removing `callBase` here feels dangerous. This may also hint to dig deeper into the hierarchy for the correct solution.

Reply:
I researched this and not found problems. The calls count of the selectionChanged event is not changed after my changes.